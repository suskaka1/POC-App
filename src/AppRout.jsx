import React from 'react';
import {
    BrowserRouter as Router,
    Route,
    Switch
} from "react-router-dom";
import Login from './Login';
import Welcome from './Welcome';

const AppRoute = () =>{
    return(
        <Router>
            <Switch>
                <Route exact path='/' component={Login} />
                <Route exact path='/Welcome' component={Welcome} />
            </Switch>
        </Router>
    )
}
export default AppRoute;