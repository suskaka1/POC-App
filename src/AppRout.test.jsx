import React from 'react';
import { shallow } from 'enzyme';
import AppRout from './AppRout';

describe("AppRout", () => {
  it("should render my component", () => {
    const wrapper = shallow(<AppRout />);
    expect(wrapper).toMatchSnapshot();
  });
});