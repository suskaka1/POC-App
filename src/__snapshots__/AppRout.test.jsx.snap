// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRout should render my component 1`] = `ShallowWrapper {}`;
